lib/livebook_web/live/settings_live.ex

```diff
@@ -124,6 +140,50 @@ defmodule LivebookWeb.SettingsLive do
     """
   end
 
+  defp autosave_path_select(assigns) do
+    ~H"""
+    <div class="flex items-center justify-between p-4">
```

Comment:
I'd remove the spacing, and we probably don't need the container altogether.

```suggestion
```

Btw. then you can split the rendering with pattern matching, like

```elixir
defp autosave_path_select(%{state: %{dialog_open?: true}} = assigns)
```

Reply:
Yup, looks much cleaner now :)